packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -61,8 +61,14 @@ function findOfficeApiReferencesInScope(scope: Scope): void {
       reference.resolved &&
       proxyVariables.has(reference.resolved)
     ) {
-      if (isLoadReference(reference.identifier)) {
+      const node: TSESTree.Node = reference.identifier;
+      if (isLoadReference(node)) {
         apiReferences.push({ operation: "Load", reference: reference });
+      } else if (
+        node.parent?.type === TSESTree.AST_NODE_TYPES.MemberExpression &&
+        isCallingMethod(findTopLevelExpression(node.parent))
```

Comment:
I would do the findTopLevelExpression call inside the 'isCallingMethod' method and just pass in the node.